detect_secrets/plugins/common/initialize.py

```diff
@@ -30,6 +30,31 @@ def from_parser_builder(plugins_dict):
     return tuple(output)
 
 
+def merge_plugin_from_baseline(baseline_plugins, args):
+    # if --use-all-plugins
+    #   include all parsed plugins
+    # else
+    #   include all baseline plugins
+    #   remove all disabled plugins
+
+    plugins = []
+    if args.use_all_plugins:
+        plugins = from_parser_builder(args.plugins)
+    elif args.disabled_plugins:  # strip some plugins off baseline
+        plugins = _merge_plugin_from_baseline(baseline_plugins, args)
+    else:
+        plugins = baseline_plugins
+    return plugins
+
+
+def _merge_plugin_from_baseline(baseline_plugins, args):
```

Comment:
Maybe rename to `_trim_disabled_plugins_from_baseline`, it would make the make the comments in `merge_plugin_from_baseline` less needed.